identity-service/src/index.ts

```diff
@@ -1,5 +1,12 @@
 'use strict'
 
+// Import libs before anything else becaues it takes a very long time to load.
+// Once it's imported once, it'll be in the cache and subsequent imports will be fast.
+const importLibs = async () => {
+  const { libs } = require('@audius/sdk')
+}
+importLibs()
+
```

Comment:
wow, that's interesting - how long does it take to load? Is it just as much an improvement in prod as local dev?

Reply:
Does this actually improve the build time? The point of dynamic imports is to delay the import of a large dep, but here we are just importing it immediately. I think the win with this pr is not linking libs by default, which avoids having to build libs. But I would think that the dynamic import doesn't make a difference

Reply:
good catch - since libs is needed for startup so it can't be delayed 😞 this one '@audius/sdk' import does take 15s at the very least (WITHOUT libs linked - I ran the block below to confirm) since it has like 50k lines in it. I was trying to make it import async, but I had to do `await import` for that, which gave the illusion that it started up faster since other logs came in while it imported in the background, but it ended up actually making the full init take almost twice as long. I reverted now
```
const startMs = Date.now()
const { libs: AudiusLibs } = require('@audius/sdk')
console.log(`Importing libs took ${(Date.now() - startMs) / 1000}s`)
```

I think the real solution is to have the libs dist index.js minimized or tree shake to only include what's necessary